examples/clip-selector-with-hls.html

```diff
@@ -0,0 +1,68 @@
+<html>
+<head>
+  <meta name="viewport" content="width=device-width">
+  <script src="https://cdn.jsdelivr.net/npm/hls.js@latest"></script>
+  <script type="module" src="../dist/index.js"></script>
+</head>
+<style>
+  /*
+     over-ride the opacity rule so that the media-control-bar is always
+     showing (the default behavior is to hide it when media is playing)
+   */
+  media-control-bar {
+    opacity: 1;
+  }
+</style>
+<body>
+
+<media-container>
+  <video
```

Comment:
Just want to make sure you're aware of the [hls web component](https://www.npmjs.com/package/hls-video-element).

Does the clip selector work differently with HLS than an MP4 some how?